waiter/src/waiter/util/utils.clj

```diff
@@ -111,19 +111,18 @@
 
 (defn stringify-elements
```

Comment:
Nitpick: this change belongs in a separate PR

Reply:
This happened when I deleted the timer-context printing stuff. I'll pull this out.

Reply:
#340